app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+class Like < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+
+  after_save :update_likes_counter
+
+  private
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
- [OPTIONAL] Just like I suggested above, using the `!increment` works better in this scenario. So I'll suggest you update your method to look something like this. :wave: :waffle: 
```ruby

def update_likes_counter
  post.increment!(:likes_counter)
end
```